Documentation/bootstrapping-guidelines.md

```diff
@@ -65,60 +85,106 @@ Building .NET for the first time is a two stage process:
+   See the Stage 1 instructions in [Building on a supported platform (Using RID
+   known to .NET)](#building-on-a-supported-platform-using-rid-known-to-net).
 
 **Stage 2:**
 
-1. Now you have a RID-specific SDK that knows about your new RID, build with Stage 1 SDK as done in [Building on a supported platform (Using RID known to .NET)](#building-on-a-supported-platform-using-rid-known-to-net).
+1. Now you have a RID-specific SDK that knows about your new RID, build with
+   Stage 1 SDK as done in [Building on a supported platform (Using RID known to
+   .NET)](#building-on-a-supported-platform-using-rid-known-to-net).
 
 ## Building for New Architecture (Using a RID unknown to .NET)
 
-Building for an architecture that Microsoft does not currently build the SDK for is possible but requires more work.
+Building for an architecture that Microsoft does not currently build the SDK for
+is possible but requires more work.
 
 **RIDs:**
 
-The RID graph or runtime fallback graph is a list of RIDs that are compatible with each other. You can see the list of supported RIDs and the RID graph in the [runtime.json](https://github.com/dotnet/runtime/blob/main/src/libraries/Microsoft.NETCore.Platforms/src/runtime.json). Learn more about RID catalog in [here](https://learn.microsoft.com/en-us/dotnet/core/rid-catalog#linux-rids).
+The RID graph or runtime fallback graph is a list of RIDs that are compatible
+with each other. You can see the list of supported RIDs and the RID graph in the
+[runtime.json](https://github.com/dotnet/runtime/blob/main/src/libraries/Microsoft.NETCore.Platforms/src/runtime.json).
+Learn more about RID catalog in
+[here](https://learn.microsoft.com/en-us/dotnet/core/rid-catalog#linux-rids).
 
-You will need to update the RID graph to include your new platform and runtime IDs.  See <https://github.com/dotnet/runtime/pull/82382> or <https://github.com/dotnet/runtime/pull/75396> for examples.
+You will need to update the RID graph to include your new platform and runtime
+IDs.  See <https://github.com/dotnet/runtime/pull/82382> or
+<https://github.com/dotnet/runtime/pull/75396> for examples.
 
-Building for unsupported architectures require cross-compilaton on the supported platform. Determine the compatible host to build which provides cross-compilation toolchain.  [IBM has published](https://community.ibm.com/community/user/powerdeveloper/blogs/sapana-khemkar/2023/01/13/cross-build-dotnet7-on-x86-ibm-power?CommunityKey=8cc2a1f0-6307-48cb-9178-ace50920244e) a detailed description of how they successfully built .NET 7 for IBM Power.
+Building for unsupported architectures require cross-compilaton on the supported
+platform. Determine the compatible host to build which provides
+cross-compilation toolchain.  [IBM has
+published](https://community.ibm.com/community/user/powerdeveloper/blogs/sapana-khemkar/2023/01/13/cross-build-dotnet7-on-x86-ibm-power?CommunityKey=8cc2a1f0-6307-48cb-9178-ace50920244e)
```

Comment:
Does the line limit not apply here because this is a link?

Reply:
Link urls are excluded from the limit.